docs/src/devdocs/elements.md

```diff
@@ -35,5 +35,8 @@ Ferrite.getvertexinstances
 Ferrite.filterfaces
 Ferrite.filteredges
 Ferrite.filtervertices
+Ferrite.element_to_face_transformation
+Ferrite.face_to_element_transformation
+Ferrite.InterfaceTransformation
+Ferrite.update!
```

Comment:
Is this here correct? The functon is already in the boundary condition docs page. It also feels like `update!` might clash with names from other packages, so we might not want to use this name anyway. Maybe we can be more specific? Really not sure here.